src/Schedule.h

```diff
@@ -373,6 +445,24 @@ class StageSchedule {
     std::vector<PrefetchDirective> &prefetches();
     // @}
 
+    /** Until which loop level (starting from outermost) we should fuse
+     * computation of this function stage with another function stage? The
+     * function we are fusing this function with and this function should
+     * be independent of each other. See \ref Func::compute_with and
```

Comment:
What does "independent of each other" mean? They're not required to be distinct are they?